stdlib/format.mli

```diff
@@ -1120,6 +1120,14 @@ val pp_print_result :
 
     @since 4.08 *)
 
+val pp_print_either :
+  left:(formatter -> 'a -> unit) ->
+  right:(formatter -> 'b -> unit) -> formatter -> ('a, 'b) Either.t -> unit
+(** [pp_print_either ~left ~right ppf e] prints [e] on [ppf] using
+    [left] if [e] is [Either.Left _] and [right] if [e] is [Either.Right _].
```

Comment:
Note: this documentation comment is consistent with the `_result` documentation above, but to be more precise it could say "using [left v] if [e] is [Either.Left v]" (and same in the other case)". I don't have strong opinions about which form is better, so leaving it as-is is also fine, please do as you think is best.

Reply:
I'm not really sure, so I'll just leave it as it is.